include/zephyr/net/net_if.h

```diff
@@ -2279,6 +2279,15 @@ struct in_addr *net_if_ipv4_get_ll(struct net_if *iface,
 struct in_addr *net_if_ipv4_get_global_addr(struct net_if *iface,
 					    enum net_addr_state addr_state);
 
+/**
+ * @brief Get IPv4 netmask of an interface.
+ *
+ * @param iface Interface to use.
+ *
+ * @return A pointer to the netmask set on the interface, NULL if not found.
+ */
+struct in_addr *net_if_ipv4_get_netmask(struct net_if *iface);
```

Comment:
Just wondering here that should we just return the in_addr which is basically a uint32_t type variable. Will the caller of this function need a pointer the original value?

Reply:
I don't need a pointer for the DHCP case, maybe indeed it makes more sense to return a value here, changed.